src/core/src/energy/global/wolf.rs

```diff
@@ -193,11 +197,20 @@ impl GlobalPotential for Wolf {
     }
 
     fn forces(&self, system: &System) -> Vec<Vector3D> {
+        // To avoid race conditions, each thread needs its
+        // own local forces Vec
+        let thread_forces_mutex : CachedThreadLocal<Mutex<Vec<Vector3D>>> = CachedThreadLocal::new();
```

Comment:
`CachedThreadLocal` only returns immutable references so we need interior mutability, since it's in a parallel context I went with a `Mutex`, maybe a `RefCell` is enough I'll check that.